react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js

```diff
@@ -180,7 +189,22 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
 
         return dontShowAgain
         || this.state.closePressed
-        || !this.state.shouldShow;
+        || !this.state.shouldShow
+        || this.props.iAmRecorder;
+    }
+
+    /**
+     * Decides whether the chrome extension banner is enabled based on config and environment.
+     *
+     * @returns {boolean}
+     * @protected
+     */
+    _isBannerEnabled() {
+        return typeof interfaceConfig !== 'undefined'
+            && interfaceConfig.SHOW_CHROME_EXTENSION_BANNER
+            && browser.isChrome()
+            && Platform.OS !== 'android'
```

Comment:
we haave some `_isMobileBrowser` in base/environment, probably worth it to expose it and use it here.

Reply:
I exposed it and used it in various places where this check was needed.